docs/reference/installation/_interfaces/apt.cue

```diff
@@ -23,7 +23,6 @@ installation: _interfaces: apt: {
 
 	role_implementations: [string]: {
 		commands: role_implementations._systemd_commands & {
-			_config_path: paths.config
```

Comment:
Dropped this since we can reference the parent definition now that commands is nested in `#Interface`